tests/unit/protobuf/test_protobuf_compatibility.py

```diff
@@ -300,3 +300,37 @@ def test_compatibility_ordering_change2():
     assert result.is_compatible()
     assert len(result.result) == 1
     assert result.result[0].modification == Modification.FIELD_ADD
+
+
+def test_compatibility_field_tag_change():
+    self_schema = """\
+syntax = "proto3";
+package pkg;
+message Foo {
+    string fieldA = 1;
+    string fieldB = 2;
+    string fieldC = 3;
+    string fieldX = 4;
+}
+"""
+
+    other_schema = """\
+syntax = "proto3";
+package pkg;
+message Foo {
+    string fieldA = 1;
+    string fieldB = 2;
+    string fieldC = 3;
+    string fieldD = 4;
+    string fieldX = 5;
+}
+"""
+
+    self_parsed: ProtoFileElement = ProtoParser.parse(location, self_schema)
+    other_parsed: ProtoFileElement = ProtoParser.parse(location, other_schema)
+
+    result = CompareResult()
+    self_parsed.compare(other_parsed, result)
+    assert not result.is_compatible()
+    assert result.result[0].modification == Modification.FIELD_TAG_ALTER
```

Comment:
can we assert also that the number of modifications its only 1? like: `assert len(result.result) == 1`